src/main/scala/org/virtuslab/beholder/filters/json/JsonFilterFieldsComponent.scala

```diff
@@ -18,71 +16,98 @@ trait JsonFilterFieldsComponent extends FilterFieldComponent with SeqParametersH
   import unicorn.profile.api._
   import CustomTypeMappers._
 
-  abstract class JsonFilterField[A: TypedType, B] extends MappedFilterField[A, B] {
-    def fieldTypeDefinition: JsValue
-
-    protected def filterFormat: Format[B]
-
-    protected[json] def valueWrite: Writes[A]
-
-    final def writeValue(value: Any): JsValue = valueWrite.writes(value.asInstanceOf[A])
+  object SingleFieldTypeDefinition {
```

Comment:
shouldn't scope of this helper object be somehow limited?

Reply:
right, done